packages/react-databrowser/src/components/databrowser/components/add-data/add-data-dialog.tsx

```diff
@@ -56,7 +56,7 @@ export function AddDataDialog({ onNewDataAdd }: Props) {
       const exp = Number(formData.get("exp"));
       const expUnit = formData.get("exp-unit") as ExpUnitUnion | undefined;
       const ttl = expUnit ? convertToSeconds(expUnit, exp) : null;
-      const ok = await addData.mutateAsync([key, value, ttl]);
+      const ok = await addData.mutateAsync([key, value, ttl, false]);
```

Comment:
why do we add false here?

Reply:
It was giving a type error because it was left as undefined, it's practically the same thing